.github/workflows/ci.yml

```diff
@@ -1,9 +1,6 @@
 name: CI
 
-on:
-  push:
-    branches: [ master ]
-  pull_request:
+on: [push, pull_request]
```

Comment:
Why did you change this? Running actions for every push would increase the number of spent minutes.